.cirrus.yml

```diff
@@ -9,22 +9,22 @@ env:
   CC_MAPS_API_KEY: ENCRYPTED[!c95bfbac40f5fe9f371207ecfc51501f94132a3c441df1c40175e7391c20a3be3ad398671f4df6f4014b99065ad257b1!]
   GOOGLE_KEY_API : ENCRYPTED[a07be5ba98879121ccec837d5d70c321c57a7e6aa82022d4be5047ff86737d538abb54ad43a63efec2a1a07359b01b60]
 
-check_gitlint_task:
```

Comment:
Why do you have commented ?

Reply:
Gitlint is somehow always failing, even if i have compliant commit messages, that's why i deactivated it

Reply:
Do you mean to reactivate it ?

Reply:
No, i need to figure out why it's failing first. But for now, I would say leave it like this.